sdk/storage/azure-storage-common/test/ut/test_base.hpp

```diff
@@ -0,0 +1,211 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/platform.hpp>
+
+#include <chrono>
+#include <cstdint>
+#include <limits>
+#include <vector>
+
+#include <azure/core/base64.hpp>
+#include <azure/core/datetime.hpp>
+#include <azure/core/etag.hpp>
+#include <azure/core/io/body_stream.hpp>
+#include <azure/core/test/test_base.hpp>
+#include <azure/identity/client_secret_credential.hpp>
+#include <azure/storage/blobs.hpp>
+#include <azure/storage/common/internal/constants.hpp>
+#include <azure/storage/common/storage_common.hpp>
+#include <gtest/gtest.h>
+
+namespace Azure { namespace Storage {
+
+  namespace Test {
+
+    class StorageTest : public Azure::Core::Test::TestBase {
+    protected:
+      const std::string& StandardStorageConnectionString();
+      const std::string& PremiumStorageConnectionString();
+      const std::string& BlobStorageConnectionString();
+      const std::string& PremiumFileConnectionString();
+      const std::string& AdlsGen2ConnectionString();
+      const std::string& AadTenantId();
+      const std::string& AadClientId();
+      const std::string& AadClientSecret();
+
+      std::string GetContainerValidName() const
+      {
+        std::string name(m_testContext.GetTestSuiteName() + m_testContext.GetTestName());
+        // Make sure the name is less than 63 characters long
+        auto const nameSize = name.size();
+        size_t const maxContainerNameSize = 63;
+        if (nameSize > maxContainerNameSize)
```

Comment:
In storage code, we always use `const type` rather than `type const`. (Like from Line 30 to Line 37 in this file.) We should be consistent.

ditto for other places